Update/some_004.txt

```diff
@@ -1620,20 +1620,20 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fda762>千紗登</color>", NULL, "<color=#fda762>Chisato</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300265", 256, TRUE);
 	OutputLine(NULL, "「しっかし、あの尾形も容赦がないよね。",
-			NULL, "\"Actually, Ogata is pretty unforgiving.", Line_WaitForInput);
+			NULL, "\"Sheesh, Ogata doesn't have an ounce of mercy, does she?", Line_WaitForInput);
 	ModPlayVoiceLS(4, 37, "ps3/s04/37/310300266", 256, TRUE);
 	OutputLine(NULL, "もうちょっと言い方ってもんがあるだろうにさ」",
-			NULL, " I have a little more to say to her.\"", Line_Normal);
+			NULL, " She could really work on her bedside manner.\"", Line_Normal);
 	ClearMessage();
 
 //珠子rvS04/38/310400109.「まぁ、あの先生はハッパかけるのが教師の役目だって、信じてるところがあるからね。kvS04/38/310400110.私たちは高校受験前の塾とかで慣れっこだったから、別にいいんだけどさ」
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6f6599>珠子</color>", NULL, "<color=#6f6599>Tamako</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400109", 256, TRUE);
 	OutputLine(NULL, "「まぁ、あの先生はハッパかけるのが教師の役目だって、信じてるところがあるからね。",
-			NULL, "\"Well, I believe the teacher is just trying to fill her role as an instructor.", Line_WaitForInput);
+			NULL, "\"Well, she probably believes that a teacher needs to be harsh to motivate their students.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 38, "ps3/s04/38/310400110", 256, TRUE);
 	OutputLine(NULL, "私たちは高校受験前の塾とかで慣れっこだったから、別にいいんだけどさ」",
-			NULL, " We were prepared for the possibility of cram school before college entrance exams, so this will be ok.\"", Line_Normal);
+			NULL, " Though we were used to that sort of thing from the cram school and like we took before our high school entrance exams, so we could take it.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " Though we were used to that sort of thing since we dealt with cram school and such before our high school entrance exams, so we could take it.\"", Line_Normal);
```